tests/affine-cloud-copilot/e2e/utils/settings-panel-utils.ts

```
@@ -0,0 +1,172 @@
+      .getByTestId(WORKSPACE_EMBEDDING_SWITCH_TEST_ID)
+      .locator('input');
+    await expect(input).toHaveAttribute('value', enabled ? 'on' : 'off');
+  }
+
+  public static async toggleWorkspaceEmbedding(page: Page) {
+    const input = await page.getByTestId(WORKSPACE_EMBEDDING_SWITCH_TEST_ID);
+    await input.click();
+  }
+
+  public static async enableWorkspaceEmbedding(page: Page) {
+    const enabled = await this.isWorkspaceEmbeddingEnabled(page);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)